#External Prior/Transform Input
extrinsics:
  mapFrame: map
  odomFrame: odom
  lidarFrame: crl_rzr/base_link                     #LiDAR frame name - used to lookup LiDAR-to-ExternalSensor Frame
  imuFrame: crl_rzr/xsens_link                 #Frame of IMU used in integrator - Used for lookup transform to lidarFrame
  leftGnssFrame: GNSS_L
  rightGnssFrame: GNSS_R